packaging/suse/trento-server/values-overwrite.yaml

```diff
@@ -2,9 +2,9 @@ trento-web:
   image:
     repository: registry.suse.com/trento/trento-web
 
-trento-runner:
+trento-wanda:
```

Comment:
Ok, I implemented this exactly in other pr :joy:

Reply:
ah ok i will rebase